seq-mainnet-1/README.md

```diff
@@ -36,392 +27,19 @@ Alternate public nodes:
-	"commission-rate": "0.05",
-	"commission-max-rate": "<MAX-RATE>",
-	"commission-max-change-rate": "<MAX-CHANGE-RATE>",
-	"min-self-delegation": "1"
-}
-```
-
-...where the pubkey can be obtained using `fuelsequencerd tendermint show-validator`.
-
-### What to Expect
-
-If the genesis time is in the future, the Sequencer will show:
-
-```
-Genesis time is in the future. Sleeping until then... genTime=YYYY-MM-DDThh:mm:ssZ module=server
-```
-
-In the same situation, the Sidecar will show:
-
-```
-did not find a start block, but maybe Sequencer hasn't started; ensure Sequencer is available at <IP:PORT>
-```
-
-If the genesis time is in the past, then the Sequencer should show block generation, and the Sidecar should show block extraction.
-
-### Tendermint KMS
-
-If you will be using `tmkms`, make sure that in the config:
-
-- Chain ID is set to `seq-mainnet-1` wherever applicable
-- `account_key_prefix = "fuelsequencerpub"`
-- `consensus_key_prefix = "fuelsequencervalconspub"`
-- `sign_extensions = true`
-- `protocol_version = "v0.34"`
-
-### Additional Advanced Configuration
-
-Sidecar flags:
-
-- `development`: starts the sidecar in development mode
-- `eth_max_block_range`: max number of Ethereum blocks queried at one go
-- `eth_min_logs_query_interval`: minimum wait between successive queries for logs
-- `unsafe_eth_start_block`: the Ethereum block to start querying from
-- `unsafe_eth_end_block`: the last Ethereum block to query. Incorrect use can cause the validator to propose empty blocks, leading to slashing!
-- `sequencer_path_to_cert_file`: path to the certificate file of the Sequencer infrastructure for secure communication. Specify this value if the Sequencer infrastructure was set up using TLS.
-- `sidecar_path_to_cert_file`: path to the certificate file of the sidecar server for secure communication. Specify this value if you want to set up a sidecar server with TLS.
-- `sidecar_path_to_key_file`: path to the private key file of the sidecar server for secure communication. Specify this value if you want to set up a sidecar server with TLS.
-- `prometheus_enabled`: enables serving of prometheus metrics
-- `prometheus_listen_address`: address to listen for prometheus collectors (default ":8081")
-- `prometheus_max_open_connections`: max number of simultaneous connections (default 3)
-- `prometheus_namespace`: instrumentation namespace (default "sidecar")
-- `prometheus_read_header_timeout`: amount of time allowed to read request headers (default 10s)
-- `prometheus_write_timeout`: maximum duration before timing out writes of the response (default 10s)
-
-Sidecar client flags:
-
-- `sidecar_grpc_url`: the sidecar's gRPC endpoint
-- `query_timeout`: how long to wait before the request times out
+The first steps for both cases are the same, and can be found in the [Run Sequencer Node](./RUN_NODE.md) document. Afterward, to continue the steps towards running a validator, refer to the [Run Sequencer Validator](./RUN_VALIDATOR.md) document.
```

Comment:
I think I would combine `RUN_NODE.md` and `RUN_VALIDATOR.md` into a single document with headings `Run Full Node` and `Run Validator`. This will make the guide more user friendly because currently validator operators will have to go through different guides to have everything up and running.